protocol/x/clob/keeper/keeper.go

```diff
@@ -44,8 +44,9 @@ type (
 
 		memStoreInitialized *atomic.Bool
 
-		MaxLiquidationOrdersPerBlock    uint32
-		MaxDeleveragingAttemptsPerBlock uint32
+		MaxLiquidationOrdersPerBlock        uint32
+		MaxDeleveragingAttemptsPerBlock     uint32
+		MaxDeleveragingSubaccountsToIterate uint32
```

Comment:
Nit: Store these as a single `Flags ClobFlags` struct so that they don't have to manually be updated if we add more flags